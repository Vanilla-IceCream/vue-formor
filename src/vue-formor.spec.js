import { mount } from '@vue/test-utils';

import Form from './Form.vue';

process.chdir(__dirname);

describe('vue-formor', () => {
  it('Form', () => {
    const wrapper = mount(Form, {});

    expect(wrapper.html()).toMatchSnapshot();
  });
});
