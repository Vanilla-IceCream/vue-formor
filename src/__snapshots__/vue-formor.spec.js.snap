// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vue-formor Form 1`] = `
"<form>
  <div class=\\"font-bold\\">Form</div>
  <div><label for=\\"form-firstField\\">First Field:</label><input id=\\"form-firstField\\" type=\\"text\\" class=\\"border\\">
    <div class=\\"text-red-500\\"></div>
  </div>
  <div><label for=\\"form-secondField\\">Second Field:</label><input id=\\"form-secondField\\" type=\\"text\\" class=\\"border\\">
    <div class=\\"text-red-500\\"></div>
  </div>
  <div><button id=\\"submit\\" type=\\"button\\" class=\\"bg-blue-500 hover:bg-blue-700 text-white px-4 py-1 rounded\\"> Submit </button></div>
</form>"
`;
